src/indexer.ts

```diff
@@ -221,6 +226,7 @@ export class SystemVerilogIndexer {
         @return number of added files
     */
     addDocumentSymbols(document: TextDocument, symbolsMap: Map<string, Array<SystemVerilogSymbol>>): Thenable<number> {
+        // eslint-disable-next-line no-async-promise-executor
         return new Promise(async (resolve) => {
```

Comment:
I think the `async` should be removed from this function call instead of being waived.

Reply:
When I remove this async one indexer test fails. Left it as is.